src/bin/upgrade_octree.rs

```diff
@@ -42,6 +42,30 @@ fn upgrade_version9(directory: &Path, mut meta: proto::Meta) {
     meta.write_to_writer(&mut buf_writer).unwrap();
```

Comment:
Should you call `upgrade_version10()` here to go directly from 9 -> 11 instead of upgrading to the outdated version 10?

Reply:
The loop below is taking care of this.

Reply:
ah, I missed that 👍